app/components/CleanComponent/CleanComponent.js

```diff
@@ -0,0 +1,142 @@
+// @flow
+import React, { Component } from "../../../AppData/Local/Microsoft/TypeScript/2.9/node_modules/@types/react";
```

Comment:
Why is there two of this file?

Reply:
Left over from when I originally thought I would dedicate a seperate component to the Clean interface. Removed